packages/command/messages/command.json

```diff
@@ -1,6 +1,6 @@
 {
-  "RequiresUsernameError": "This command requires a scratch org username set either with a flag or by default in the config.",
-  "RequiresDevhubUsernameError": "This command requires a dev hub org username set either with a flag or by default in the config.",
+  "RequiresUsernameError": "This command requires a username. Specify it with the -u parameter or by \"sfdx config:set defaultusername=<username>\".",
```

Comment:
small change:  "... or **with the** \"sfdx config:set defaultusrname=<username>\" **command**. "   (i added bold to show suggested additions)